spec.md

```diff
@@ -840,6 +841,21 @@ If the image had already been deleted or did not exist, a `404 Not Found` respon
 
 > for more details, see: [compatibility.md](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#backward-compatibility)
 
+### Registry Proxying
+
+A registry MAY operate as a proxy to another registry to support pull-through caching or proxying of pull operations (such as fetching tags, manifests, or blobs).
```

Comment:
I've broken this down into 3 separate sentences that articulate one clear thought. I think this also makes it more obvious that this feature could be used for more than the examples provided.

>A registry MAY operate as a proxy to another registry to delegate functionality or implement additional functionality.
>An example of delegating functionality is proxying pull operations to another registry.
>An example of adding functionality is implementing a pull through cache of pulls to another registry.

Reply:
Thanks, agreed, much clearer